src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = { new Truck(), new Bulldozer(), new Excavator() };
```

Comment:
Consider using abstract references instead of specific ones where possible. According to the checklist, it's better to use `Machine` as the type for the elements in the `machines` array to allow for polymorphism.